CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+Notable changes to Draft.js will be documented in this file.
+
+Changes to `src` are live in production on facebook.com at the time of release.
+
+## 0.2.0 (March 8, 2016)
+
+### Added
+
+* Move `white-space: pre-wrap` into inline style to resolve insertion issues
+* `handleDrop` prop method for `Editor` to allow manual drop management
+* `decoratedText` prop for decorator components
+* `getVisibleSelectionRect`, to provide Rect for DOM selection
+* Create Slack team inviter
```

Comment:
I don't normally note non-source changes in changelogs but if you want to this also seems fine.